src/main/resources/log4j.properties

```diff
@@ -0,0 +1,7 @@
+# This sets the global logging level and specifies the appenders
```

Comment:
we need it? I think the host should decide on these options no?

Reply:
not sure what you mean, but in order log4j will printout something you have to have this file with some kind of default.

otherwise it will not printout anything

Reply:
fixed